src/ucp/core/ucp_worker.c

```diff
@@ -3105,11 +3102,11 @@ static int ucp_worker_do_ep_keepalive(ucp_worker_h worker, ucs_time_t now)
     if (status != UCS_OK) {
         ucs_diag("ep %p lane[%d]=%p: failed to send keepalive: %s", ep, lane,
                  ep->uct_eps[lane], ucs_status_string(status));
+    } else {
+        ucs_trace("worker %p: keepalive done on ep %p, now: <%lf sec>", worker, ep,
```

Comment:
maybe we could align to format of messages in line 3103 and line 3106

Reply:
done